src/computation/CopyComputation.cpp

```diff
@@ -57,7 +58,7 @@ void CopyComputation::performComputation(std::shared_ptr<wrench::ActionExecutor>
                     "Reading file " + fs.first->getID() + " finished before it started!");
         }
     }
-    if (!(std::abs(data_size - total_data_size) < 1.)) {
+    if (data_size > total_data_size) {
```

Comment:
Wouldn't be a check for strictly equal here better?

Reply:
Oh right. So something like "if (data_size != total_data_size)  then throw"?